### ISSUE TITLE
populate repo

### ISSUE THREAD

**Issue by bparees** [openshift-s2i/s2i-nodejs-community#1]
need to populate this repo from https://github.com/ryanj/s2i-nodejs-container

@ryanj unfortunately we didn't get to finish our email conversation, but is there a reason you wouldn't be ok with simply populating this repo and then forking it (and deleting your original repo) so you don't have to maintain changes in two personal repos?

**Comment by bparees** [comment]
@ryanj bump

**Comment by ryanj** [comment]
Hi @bparees!  
A few questions for you:
- [ ] Do you have anyone available to merge PRs and keep the repo up to date as changes are made?
- [ ] Are you able to offer any build automation support (generate new images on change)?
- [ ] Are you interested in hosting the resulting images under an official account?

If I'm responsible for building and hosting the resulting images, I'd like to have commit access to the related sources in order to keep them updated and relevant.

Feel free to hit me up on kubernetes.slack.com to discuss in more detail.

**Comment by bparees** [comment]
>  Do you have anyone available to merge PRs and keep the repo up to date as changes are made?

yes, we've got a couple folks who keep an eye on the community repos and can merge.  @jawnsy is the other main person, and he's working on a process that will (if approved) also provide a path to give commit access to the original contributor of the repo.

> Are you able to offer any build automation support (generate new images on change)?

also being discussed in @jawnsy's doc.  I'm less optimistic about this one.  I suspect the best we might be able to do (and even this is debatable) is setup dockerhub automated image builds off the github repo.  Of course that's something anyone can do in their dockerhub org, too.  But that doesn't provide PR testing or even post-merge testing.

> Are you interested in hosting the resulting images under an official account?

under discussion, so far we do not host built images of anything in openshift-s2i under a redhat/openshift related dockerhub org, with the exception of wildfly because that one is maintained by me.  Again there are some hurdles we need to work through if we're going to make images available under a redhat/openshift org which i'm not sure we're prepared to surmount, in terms of careful auditing of what is going into those images.  (eg if someone submits a PR to update the dockerfile to curl a new file, are we going to go make sure that file isn't malware?  if we aren't going to do that, are we really going to want to publish that image under an official org on dockerhub? probably not)

> If I'm responsible for building and hosting the resulting images, I'd like to have commit access to the related sources in order to keep them updated and relevant.

that seems pretty reasonable to me and i think @jawnsy is intending that be one of the things the openshift-s2i community will offer contributors, but we need to run it up a little higher to make sure we're ok doing so.
